packages/xml-views-completion/src/providers/elementName/classes.ts

```diff
@@ -156,7 +156,19 @@ function handleDefaultAggregationScenario({
   parentXMLElement: XMLElement;
   parentUI5Class: UI5Class;
 }): classSuggestionContext | null {
-  const defaultAggregation = parentUI5Class.defaultAggregation;
+  let defaultAggregation: UI5Aggregation | undefined = undefined;
+
+  // handle inheritance
+  let currentClass: UI5Class | undefined = parentUI5Class;
+  while (currentClass) {
```

Comment:
@petermuessig : This seems to make sense
Can you verify that this is the correct behavior in UI5?